Wikipedia/Code/NSURL+WMFLinkParsing.m

```diff
@@ -23,6 +23,23 @@ + (NSURL *)wmf_URLWithDefaultSiteAndLanguageCode:(nullable NSString *)languageCo
     return [self wmf_URLWithDomain:WMFConfiguration.current.defaultSiteDomain languageCode:languageCode];
 }
 
++ (NSURL *)wmf_URLWithDefaultSiteAndContentLanguageCode:(nullable NSString *)contentLangaugeCode {
```

Comment:
Tiny spelling thing - contentLangaugeCode > contentLanguageCode.

Reply:
This has been removed.